additional-databases/fuseki/README.md

```diff
@@ -0,0 +1,25 @@
+## Connecting graph notebook to local Fuseki Endpoint
+
+A quite lightweight SPARQL endpoint to play locally is [Apache Fuseki](https://jena.apache.org/documentation/fuseki2/index.html).
+
+Fuseki provides separate endpoints for each configured dataset. 
+For instance, if a dataset named `ds` has been defined in Fuseki, the corresponding SPARQL endpoint will be available from `http://localhost:3030/ds/sparql`.
+
+Thus, it is necessary to define the name of the dataset in the `sparql.endpoint_prefix` part of the configuration,
+like in the following example that connect to `http://localhost:3030/ds/sparql`:
+
+```
+%%graph_notebook_config
+{
+  "host": "localhost",
+  "port": 3030,
+  "auth_mode": "DEFAULT",
+  "iam_credentials_provider_type": "ROLE",
+  "load_from_s3_arn": "",
+  "ssl": false,
+  "aws_region": "",
+  "sparql": {
+    "endpoint_prefix": "ds"
```

Comment:
We have changed this mechanism recently to allow the full api path to be specified instead of assuming that it ends in `/sparql`. For instance, the following would now be the equivalent:

```
%%graph_notebook_config
{
  "host": "localhost",
  "port": 3030,
  "auth_mode": "DEFAULT",
  "iam_credentials_provider_type": "ROLE",
  "load_from_s3_arn": "",
  "ssl": false,
  "aws_region": "",
  "sparql": {
    "path": "ds/sparql"
  }
}
```